### ISSUE TITLE
Content shown as free, charged LBC

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-app#344]
## The Issue
We've had a couple reports of users seeing the It's a Disaster video as Free, when in reality it costs 7-9 LBC (depending on LBC price).  When a user tries to download, they are still charged the LBC.  I've seen this happen on my own machine too, but not sure what exactly caused it.  When discussed within the team, it's most likely a caching issue. One time I did get an exchange error, so that may be related also.  When going to settings > Refresh the Cache, the price then shows correctly.

@6ea86b96 is aware of problem but we wanted to log it officially. 

### Steps to reproduce
1. Load LBRY
2. It's a Disaster shown as free
3. Download it, but get charged LBC

### Expected behaviour
Should not show as free, but even if it did, it should double check the price before downloading

### Actual behaviour
Content shown as free, but charged LBC when downloaded.

## System Configuration

 

- LBRY Daemon version: 0.13.1
- LBRY App version: 0.13
- LBRY Installation ID:
- Operating system: Windows

## Anything Else

```shell
2017-07-05 16:15:48,789 WARNING  lbrynet.lbrynet_daemon.ExchangeRateManager:69: There was a problem updating BTCLBC exchange rate information from lbry.io
2017-07-05 16:15:48,795 WARNING  lbrynet.lbrynet_daemon.ExchangeRateManager:69: There was a problem updating USDBTC exchange rate information from Coinbase via Google finance
```

## Screenshots

![disaster free](https://user-images.githubusercontent.com/8120721/28010805-159f2752-652e-11e7-9cbf-55adfd7e0288.PNG)

**Comment by zestyr** [comment]
I was playing around with this a few days ago and when I refreshed the cache, it showed correctly, but if I restarted the app and refreshed again (or the other way around, don't remember), it showed as `free`. I'll see if I can reproduce this, but something is definitely going on here.

**Comment by tzarebczan** [comment]
@hackrush01 saw this happen today on his discover page.

**Comment by btzr-io** [comment]
i can't recreate the issue :disappointed:, 
can some one check if #465 can fix this?

**Comment by tzarebczan** [comment]
@btzr-io I think there was also a larger formatting issue that was causing this, it should be fixed in 0.15 daemon

**Comment by tzarebczan** [comment]
Closing this as it has not happened for a while. There's a different issue with price not being shown correctly for updated claims.
